packages/okta-auth-js/lib/token.js

```diff
@@ -690,6 +692,11 @@ function renewToken(sdk, token) {
     authorizeUrl: token.authorizeUrl,
     userinfoUrl: token.userinfoUrl,
     issuer: token.issuer
+  })
+  .then(function(res) {
+    // Multiple tokens may have come back. Return only the token which was requested.
+    var tokens = res.tokens;
+    return token.idToken ? tokens.idToken : tokens.accessToken;
```

Comment:
highly subjective, but had you considered `return token.idToken || token.accessToken;`?

Reply:
This is a little subtle. `token` is the original (old) token. Based on whether the original token was an id token or an access token we want to return the corresponding (fresh) token from the hash.

Reply:
token/tokens.  Good call.